pkg/cmd/clustertask/delete.go

```diff
@@ -60,11 +81,19 @@ or
 				Err: cmd.OutOrStderr(),
 			}
 
-			if err := opts.CheckOptions(s, args, ""); err != nil {
+			availableCts, errs := ctExists(args, p)
```

Comment:
cant we just print the error of non-existent one here

because in the end if somehow the delete call caused an error in deleting, these errors will be missed.

Reply:
@piyush-garg I tried to make it similar to how kubectl does. Moreover, I tried the approach which you said but in that case it wasn't returning the error.

> because in the end if somehow the delete call caused an error in deleting, these errors will be missed.

🤔 I am unable to visualize any such. Can you please help me here :sweat_smile:. Moreover, I think if somebody doesn't have permission to delete a resource from cluster then that error message which should be displayed first showing `not able to delete` instead of first showing that `resource not found`. WDYT?

Reply:
there can be scenario like user have read right, not write rights. So in first check there will be no error

Reply:
In think in this case letting the user know that they won't be able to delete is a better idea instead of first letting them know that a particular resource is not found and then showing the error of won't be able to delete because it will not be a proper UX. 
cc: @piyush-garg @chmouel @vdemeester @vinamra28